### DISCUSSION TITLE
Can screen readers read the clr-control-error ?

### DISCUSSION THREAD

**Discussion by ChrysaLa** [vmware-clarity/ng-clarity#790] [Q&A]
I am trying to make screen readers to read content of clr-control-error but i can't. I tried something like `<clr-control-error
        tabindex="0"
        role="alert"
        aria-live="assertive"
        aria-label="Helper message container"
        >Enter numeric value</clr-control-error>` but still VoiceOver can't read it. 
        
        I would appreciate some help.

**Comment by kevinbuhmann** [comment]
On https://storybook.clarity.design/?path=/story/forms-reactive--default, Voice Over announces "more content available" while focused on an input. If you press Cmd-Control-Option-Slash as instructed, it will open the "more content menu" that can be used to read the helper text.

That said, the `aria-label` attribute in your code example will break it. It doesn't seem to read the `aria-label` at all, but you don't need it anyway because the helper text element has text content. I am not sure about the `aria-live`, though. It's probably not needed either.

**Reply by ChrysaLa** [reply]
Hello!
Thanks for your immediate reply. I tried also Cmd-Control-Option-Slash but still VoiceOver can't read clr-control-error  content.

**Reply by kevinbuhmann** [reply]
Did you remove the `aria-live` and `aria-label` attributes?

**Reply by ChrysaLa** [reply]
Yes

**Reply by kevinbuhmann** [reply]
Does it work for you on the page I linked above?

Also, I think you should remove the `role` and `tabindex` attributes as well.

**Reply by ChrysaLa** [reply]
Yes i tried that on storybook and worked as expected. I also removed all unnecessary labels but still doesn't work.The element now is like that `<clr-control-error *ngIf="form.get('myControl').errors" >Enter numeric value</clr-control-error>`. The error displayed as expected but can't be read by VoiceOver.
